ironfish-cli/src/utils/ledger.ts

```diff
@@ -257,131 +239,57 @@ export class LedgerDkg {
   }
 
   dkgRestoreKeys = async (encryptedKeys: string): Promise<void> => {
-    if (!this.app) {
-      throw new Error('Connect to Ledger first')
-    }
-
     this.logger.log('Please approve the request on your ledger device.')
 
     await this.tryInstruction((app) => app.dkgRestoreKeys(encryptedKeys))
   }
 }
 
-export class Ledger {
-  app: IronfishApp | undefined
-  logger: Logger
-  PATH = "m/44'/1338'/0"
-
+export class Ledger extends LedgerBase {
```

Comment:
nit: should be differentiated that this is the single signer some how so it's not confused with LedgerBase.